Spokenizer.py

```diff
@@ -0,0 +1,42 @@
+from Posterior import *
```

Comment:
ok, so I would remove this line... and then in posterior.py say from Spokenizer import *- and that would switch the import structure!